docs/recipes/index.rst

```diff
@@ -0,0 +1,18 @@
+Recipes
+=======
+
+The recipes are split into Appetisers_ (i.e., easier/simpler recipes -- a good place
+to start after you've been through the tutorials_), Mains_ (i.e. elaborate/advanced recipes)
+and `Local Dishes`_ (i.e. recipes for regional configurations).
+
+.. _tutorials: ../tutorials.html
+.. _Appetisers: appetisers.html
+.. _Mains: mains.html
+.. _Local Dishes: local-dishes.html
+
```

Comment:
I'm fairly sure you don't want to link directly to HTML pages here... Does it work without the `.html` extension?

Reply:
Let me try!